pipeline_dp/dp_engine.py

```diff
@@ -110,3 +114,20 @@ def unnest_cross_partition_bound_sampled_per_key(pid_pk_v):
         return self._ops.flat_map(col,
                                   unnest_cross_partition_bound_sampled_per_key,
                                   "Unnest")
+    
+    def _select_private_partitions(self, col):
+        """
+        Selects and publishes private partitions.
+
+        Args:
+            col: collection, with types for each element: 
+                (partition_key, privacy_id_count, value)
```

Comment:
sry, data format was slightly outdated, data format should be
`(partition_key, accumulator)` (src of [accumulator](https://github.com/OpenMined/PipelineDP/blob/5f8e2b5e1265af6e29f3c2fb79d0c9846a63ba43/pipeline_dp/accumulator.py#L12))

please assume that accumulator has `privacy_id_count` property (it will be added soon)